### ISSUE TITLE
BUG - Heroku - Heroku app login redirect no longer works

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#9830]
## What happened?

The [heroku web app](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Engineering/BackEnd/Testing/ApiTokens#fetching-api-tokens) - which the QA team uses to get MHV auth tokens, and use them to log into the app in staging - is no longer working. I can successfully log in with MHV credentials, but I am not redirected back to heroku at that point/can't actually use any of the heroku functionality.

We are migrating the auth slice on the front end soon (prior to the sunset of the MHV credentials in Jan 2025), so will need to be able to test with those.

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by jperk51** [comment]
@TKDickson is this only an issue for MHV login?

**Comment by TKDickson** [comment]
@jperk51 not sure. I only attempted MHV login, because that's the only one QA ever uses it for.

**Comment by TKDickson** [comment]
@jperk51 looks like the redirect still works for id.me (the only thing I have valid credentials to test), so that's at least one data point for you.![Image](https://github.com/user-attachments/assets/1d5709ec-8a29-4312-bfb7-82c9efec6992)

**Comment by jperk51** [comment]
Interesting, can you sign in through MHV on va.gov staging?

**Comment by TKDickson** [comment]
That's what happened when I attempted heroku login. I 'successfully' logged into the staging.va.gov instance of the website/that's where it dropped me when the login succeeded.

I didn't try to log into just plain staging.va.gov with MHV credentials before I shut down my Citrix session, but my guess based on what happened with heroku is that it will still work. If you need that data point before jumping into this ticket, lmk. I doubt I'll be able to check it today, but can add it to my list for next sprint.

**Comment by jperk51** [comment]
Nah, don't worry about it. We can look into it. Unfortunately based on the situation it seems highly unlikely that we will be able to fix it ourselves. Will probably need MHV or SIS to make a fix, but I also could see them saying they won't because it's going away.

**Comment by timwright12** [comment]
If this app is going to stick around we'll need to move the code over to the VA github instance
